app/lib/exchange-rate/providers/types.ts

```diff
@@ -12,5 +14,10 @@ export interface ExchangeRateProvider {
 
 export type Rates = {
   timestamp: number;
-  [ticker: Ticker]: number;
+  [ticker: Ticker]: Big;
+};
+
+export type SerializedRates = {
```

Comment:
I am thinking if it would be simpler to just have one type called rates that has ticker as string and then update the conversion method on the money to take NumberInput for rate

also there is no need to convert timestamp to string

Reply:
Okay so then we wouldn't need to serialize and deserialize? Instead the return value of getRates would already be "serialized", and convert would be doing the "deserialization"

Reply:
correct

Reply:
I removed the serialize and deserialize functions and just change the type of rates to a string